modules/store/src/store_module.ts

```diff
@@ -114,6 +114,14 @@ export interface RootStoreConfig<T, V extends Action = Action>
   runtimeChecks?: Partial<RuntimeChecks>;
 }
 
+/**
+ * A pair of reducer and name for the feature.
+ */
+export interface FeatureSlice<T, V extends Action = Action> {
+  name: string;
+  reducer: ActionReducer<T, V> | InjectionToken<ActionReducer<T, V>>;
```

Comment:
If this type is named "Slice", Is it necessary to accept an InjectionToken as `reducer`?

Reply:
No, since the user will provide both options as a "slice"